sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -276,8 +290,39 @@ export class ApplicationService extends RecordDataModelService<Application> {
       application.currentAssessment.updatedAt = now;
 
       await applicationRepository.save([application, newApplication]);
+      // Create a notification if the application is edited for the 5th time.
+      const applicationsCount = await applicationRepository.count({
+        where: { applicationNumber: newApplication.applicationNumber },
+      });
+      if (
+        applicationsCount ===
+        APPLICATION_EDIT_COUNT_TO_SEND_NOTIFICATION + 1
```

Comment:
The const `APPLICATION_EDIT_COUNT_TO_SEND_NOTIFICATION` can represent the value (including the +1) to avoid having the "+1" here.

Reply:
I feel that the currently implemented way of having the `APPLICATION_EDIT_COUNT_TO_SEND_NOTIFICATION` as per the business requirement is better since this property can be easily adjusted later on if required without diving into the code part of it where the `+1` logic is handled. IF this constant is moved out to an external properties files, then it becomes easy to update this to a new value without causing any confusion.

Reply:
If the below comment is accepted the "+1" will no longer be needed, right?
https://github.com/bcgov/SIMS/pull/3122#discussion_r1566223186